scripts/mk_genfile_common.py

```diff
@@ -550,6 +550,92 @@ def mk_z3consts_ml_internal(api_files, output_dir):
     # if VERBOSE:
     #     print ('Generated "%s/z3enums.mli"' % ('%s' % gendir))
 
+def mk_z3consts_js_internal(api_files, output_dir):
```

Comment:
seems like a copy of _mk_z3consts_py_internal".
A generalization could be a function that takes callbacks for the places where it produces different output.

Reply:
I started out with that in mind, but noticed that it was copied for every language - e.g. see `mk_z3consts_ml_internal`, `mk_z3consts_dotnet_internal`.

Reply:
ouch, it is then something for me to look at.